src/magellan_vision/scripts/detect_cones.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python2
+import rospy
+import yolo
+from geometry_msgs.msg import Point
+
+class ConeDetectNode:
+    def __init__(self):
+        # Placeholder for the eventual image publisher
+        self.image_stream_sub = rospy.Subscriber('rgb_image', Image, self._process_image)
+        # TODO: make custom message to hold cone coordinates and possibly detection score (confidence)
+        self.cone_loc_pub = rospy.Publisher('cone_loc', Point, queue_size=10)
+        self.yolo = yolo.YOLO()
+        self.coneLocations = []
+        self.coneScores = []
+
+    def _process_image(self, img):
+        (boxes, scores, classes) = self.yolo.detect_image(img)
+        for (box, score, objClass) in zip(boxes, scores, classes):
+            self.coneLocations = []
+            self.coneScores = []
```

Comment:
Recommendation: Replace these two variables with an array of tuples or `namedtuple`

Reply:
Oops, I forgot to make a gitignore with *.swp. I'll fix this tonight